app/frontend/stylesheets/components/_banner.scss

```diff
@@ -0,0 +1,37 @@
+.banner {
+  height: 90vh;
+  background-image: url("../../images/banner.jpg");
+  background-size: cover;
+  background-position: center -260px;
+}
+
+
+
+
+//----------------------nav-----------------------//
+
+
+
+
+
+//---------------------------------text-----------------------------//
+
+
+
+.text {
+  margin: 80px 0px 0px 170px;
+  h1 {
+    max-width: 280px;
+    font-size: 40px;
+    font-family: $headers-font;
+    margin-bottom: 30px;
+    color: #fff;
+  }
+
+  p{
+    font-size: 15px;
+    font-weight: bold;
+    color: #fff;
+    margin-bottom: 40px;
+  }
+}
```

Comment:
- Move this inside the .banner css selector
- Don't use fixed value margin here, use flex instead to position this
- You don't need to set the font-family to be $header-font, this is already done in the config/_setup.scss file
- Don't use margins to create distance between elements. Turn the div.text into a display flex instead and use gap
- Move the color to the .text selector, it'll apply to all children